cmd/federation.go

```diff
@@ -34,10 +35,8 @@ func main() {
 		logger.Fatal("failed connecting to queue service", err)
 	}
 	defer q.Close()
-	err = q.StartConsumer("federation")
-	if err != nil {
-		logger.Fatal("failed starting to consumer", err)
-	}
+	processActors(logger, conn, q)
+	processPosts(logger, conn, q)
```

Comment:
These probably need to be moved somewhere else _eventually_

Reply:
Yeah, that might make sense. I've been trying to keep things separate where possible (eg not have business logic in the queue code but rather have that somewhere else), but I also don't want to over-engineer this either.

Reply:
😦  That doesn't work. I took a stab at refactoring to have it run inside the queue. The problem is I need the database there also. So now I have to pass the database to the queue and that feels icky. I think I need to create another object and pass that object the db, logger, and queue (like we do the http server).

Reply:
I managed to rework things to move these commands into the queue namespace. Not sure that makes sense as a final resting place, but probably good for now